backend/package.json

```diff
@@ -12,8 +12,7 @@
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "vitest run",
-    "test:watch": "vitest",
-    "generate-client": "openapi --input ./swagger-spec.json --output ../frontend/src/api --client axios --indent 2"
```

Comment:
you removed the script. Did you document how to execute the generation now? And are you aware that this approach requires docker to generate the frontend api code?

Reply:
Done [cea7ab9](https://github.com/Oeffis/Oeffis/pull/277/commits/cea7ab9076d7a7f8fb802167311b6bf6fb39b93f)